CSS-short-description-guidelines.md

```diff
@@ -0,0 +1,18 @@
+# Guidelines for CSS short descriptions
```

Comment:
Can I suggest slightly broadening the scope of this? What do you think of this being titled "Writing short descriptions" (and renaming the file to `writing-guidelines.md` or `writing-short-descriptions.md` or something)? Seems like this might be a seed for contributing docs

```suggestion
# Writing short descriptions
```

Reply:
Thanks Daniel. I've made this change.